docs/admin/provisioners.md

```diff
@@ -101,7 +101,7 @@ will use in concert with the Helm chart for deploying the Coder server.
    secret
 
    ```shell
-   kubectl create secret generic coder-provisioner-psk --from-literal=psk=`head /dev/urandom | tr -dc A-Za-z0-9 | head -c 26`
+   kubectl create secret generic coder-provisioner-psk --from-literal=psk=$(openssl rand -base64 32)
```

Comment:
```suggestion
   kubectl create secret generic coder-provisioner-psk --from-literal=psk=`head /dev/urandom | base64 | tr -dc A-Za-z0-9 | head -c 26`
```

Reply:
running the random bytes through `base64` encoding before feeding them to `tr` seems to clear up the issue for me

Reply:
@spikecurtis yeah I'm on macOS. adding the `base64` encoding clears it up for me as well. my fish shell was not liking the backticks, but they work fine on bash.